doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -167,7 +200,7 @@ with the following sequence of :command:`vagrant` commands:
 
 .. note:: Vagrant Providers
 
-   Currently, KIWI only supports the libvirt and virtualbox providers.
+   Currently, KIWI only supports the libvirt and VirtualBox providers.
    There are also other providers available, which require a different
    box layout (currently not supported by KIWI).
```

Comment:
Maybe this note should be a moved at the beginning of the section to be more prominent?

Reply:
I've moved this up, but unfortunately when put at the top of the page then it clashes with the abstract (clash is maybe too strong, but the end result isn't pretty). 

It's now in the introductory text, but it breaks the text apart (which I also don't like).

Reply:
Ok, it was just an idea. If that doesn't work out, you can try to "merge it" with existing text, or leave it as is for the time being.

Reply:
I've merged it with the text, the box wasn't really fitting anywhere, but that information really belongs to the beginning and not to the end.